helpers/helpers.go

```diff
@@ -237,3 +238,22 @@ func Git(args ...string) {
 		os.Exit(1)
 	}
 }
+
+// RunCommand runs the given command with args and optionally prints
+func RunCommand(print bool, cmdname string, args ...string) error {
```

Comment:
Optional: make a `RunCommandSilent` and a `RunCommandPrint` (meh on the name) that just call an internal function to get rid of the bool. I mention this because of advice I got from @icarus-sparry and @cmarcelo on a similar test helper.

Reply:
I'll leave it to one of them to convince me that's more useful to do.

Reply:
The main reason is documentation. Sure your ide can tell you that the first argument for RunCommand is called 'print', but if you are reading code on github and come across 

    if err := helpers.RunCommand(false, "bsdiff", original, newfile, dotfile); err != nil {

you have to know what the parameter means. Does the false mean that cmd.Stdin should be connected to /dev/null rather than os.Stdin? In contrast

    if err := helpers.RunCommandSilent("bsdiff", original, newfile, dotfile); err != nil {

is about the same length to type (less with autocomplete), and I think is easier to understand.